CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 3.0.0
+
+* Supports Chef 13
```

Comment:
Did we drop support for older chef versions, if not, can we say "Supports chef >= X and ~> 13"